packs/device42/config.yaml

```diff
@@ -0,0 +1,5 @@
+---
+d42_server: http://127.0.0.1
+d42_username: username
+d42_password: password
```

Comment:
@bearburger it's a relatively recent change - with v1.5 we created a new way of managing pack configuration https://stackstorm.com/2016/06/30/improved-pack-configuration-user-scoped-datastore-items-secure-secrets-store/

Both the old & new methods are still supported, so there's a mix of styles out there. Eventually most of them will migrate to the new method.